tests/baselines/reference/errorForUsingPropertyOfTypeAsType03.errors.txt

```diff
@@ -19,8 +19,8 @@ tests/cases/compiler/errorForUsingPropertyOfTypeAsType03.ts(24,13): error TS2713
         type C2 = typeof Color;
     
         let a1: Color.Red.toString;
-                      ~~~
-!!! error TS2694: Namespace 'Color' has no exported member 'Red'.
+                          ~~~~~~~~
+!!! error TS2713: Cannot access 'Red.toString' because 'Red' is a type, but not a namespace. Did you mean to retrieve the type of the property 'toString' in 'Red' with 'Red["toString"]'?

```

Comment:
I think we'd really like to suggest `typeof Color.Red.toString` first if that would be valid, then fall back to using indexed access syntax.